crates/matrix-sdk/src/event_cache/linked_chunk/mod.rs

```diff
@@ -852,6 +852,12 @@ impl ChunkIdentifierGenerator {
 #[repr(transparent)]
 pub struct ChunkIdentifier(u64);
 
+impl PartialEq<u64> for ChunkIdentifier {
```

Comment:
Do we want to have this implemented generally? I understand that this is useful for tests, but perhaps we'll want to guard using a `#[cfg(test)]` attribute?

Reply:
Indeed, good point!